src/encoded/static/libs/bootstrap/tooltip.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
Personal Preference Question: Tooltip seems like a common problem. Did you find any viable react-components plugins with toooltip support. My wild guess is at least someone wrote a react-wrapper for Twitter's bootstrap tooltip.

Reply:
I’m sure, but we’ve run into _so_ many problems using other people’s React components with them not getting supported long term, and then not working with new versions of React so we always ended up having to write our own anyway. The entire React `<forms>` module used to be a third-party React module, but their support was so bad David eventually ripped it out and rewrote the whole thing himself.